implementation.js

```diff
@@ -14,29 +14,41 @@ var $toString = callBound('Object.prototype.toString');
 var stackDesc = gOPD($Error.prototype, 'stack');
 var stackGetter = stackDesc && stackDesc.get && callBind(stackDesc.get);
 
+var domExceptionClonable = !!($structuredClone && $structuredClone(new DOMException()) instanceof $Error);
```

Comment:
i think this line will throw outside a browser, or in browsers where `new DOMException()` doesn't work

Reply:
`DOMException` is supported in Node 17+, Deno, Bun, all modern browsers, and even IE9+. But I guess it's true there could be some environments that support `structuredClone` but not `DOMException`.

Reply:
ah, true, if structuredClone throws a DOMException then it's unlikely the one would exist without the other.